stream/src/main/java/com/instaclustr/kafka/connect/stream/codec/Decoders.java

```diff
@@ -0,0 +1,30 @@
+package com.instaclustr.kafka.connect.stream.codec;
+
+import com.instaclustr.kafka.connect.stream.Endpoint;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class Decoders {
+
+    public static final String DECODER_TYPE = "decoder.type";
+    public static final String PARQUET = "parquet";
+    public static final String TEXT = "text";
+    public static final ConfigDef CONFIG_DEF = new ConfigDef()
+            .define(DECODER_TYPE, ConfigDef.Type.STRING, ConfigDef.Importance.HIGH, "Decoder type");
+
+    public static Decoder<?> of(Endpoint endpoint, String filename, Map<String, String> config) throws IOException {
+        AbstractConfig validConfig = new AbstractConfig(CONFIG_DEF, config);
+        String decoderType = validConfig.getString(DECODER_TYPE).trim().toLowerCase();
+        switch (decoderType) {
+            case PARQUET:
+                return ParquetDecoder.from(endpoint.openRandomAccessInputStream(filename));
+            case TEXT:
+                return CharDecoder.of(endpoint.openInputStream(filename), config);
+            default:
```

Comment:
IIRC, we talked about adding support for CSV and JSON. 
But I am guessing the TEXT support would already cover supporting CSV? 
Will there be future work to support JSON? has it been requested by TasNetworks as well?